rules-reviewed/eap7/eap6/hibernate4.windup.xml

```diff
@@ -534,7 +517,7 @@
                 <javaclass references="org.hibernate.cfg.AnnotationConfiguration"/>
             </when>
             <perform>
-                <hint title="org.hibernate.cfg.AnnotationConfiguration was removed in Hibernate 5" effort="1" category-id="mandatory">
+                <hint title="Hibernate 5 - Removed class org.hibernate.cfg.AnnotationConfiguration" effort="1" category-id="optional">
```

Comment:
while deprecated methods could be *Optional*, package movements or class removal are not *Optional*.